### PR TITLE
add back node --experimental-repl-await to mongosh bin file

### PR DESCRIPTION
**Pull request by lrlna**
## Description
This got accidentally removed when adding eslint to cli-repl package. Adding it
back in.

### CODE REVIEW

File: packages/cli-repl/bin/mongosh.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-#!/usr/bin/env node
+#!/bin/sh
```
Comment (mcasimir) [inline packages/cli-repl/bin/mongosh.js:1]: Nice trick! Is this how it was done before?

Comment (lrlna) [reply packages/cli-repl/bin/mongosh.js:1]: yes it was!

**Comment by lrlna** [comment]
@mcasimir added a test as well, good point.

File: packages/cli-repl/test/e2e.spec.ts [hunk 2]
```diff
@@ -72,6 +72,14 @@ describe('e2e', function() {
       });
     });
 
+    it('does not throw for a repl await function', async() => {
+      shell.stdio.stdin.write('await Promise.resolve(\'Nori-cat\');');
```
Comment (mcasimir) [inline packages/cli-repl/test/e2e.spec.ts:76]: 👍 🐱

**Pull request merged**